WebService/Runtime/Config.cs

```diff
@@ -109,14 +111,17 @@ public class Config : IConfig
         private const string AZURE_RESOURCE_GROUP = DEPLOYMENT_KEY + "azure_resource_group";
         private const string AZURE_IOTHUB_NAME = DEPLOYMENT_KEY + "azure_iothub_name";
 
+        private IHttpClient httpClient;
+
         public int Port { get; }
         public ILoggingConfig LoggingConfig { get; set; }
         public IClientAuthConfig ClientAuthConfig { get; }
         public IServicesConfig ServicesConfig { get; }
         public IRateLimitingConfig RateLimitingConfig { get; set; }
         public IDeploymentConfig DeploymentConfig { get; set; }
         public IConcurrencyConfig ConcurrencyConfig { get; set; }
-
+        public IDiagnosticsLogger DiagnosticsLogger { get; set; }
```

Comment:
the `Config` class is not supposed to provide dependencies like the diagnostics logger, I think we need to remove this and the http client and the `GetLogger` method from configData. Not sure why you chose this approach, but dependencies should be provided via the DI.

Reply:
done.